packages/template/__tests__/emit-content.test.ts

```diff
@@ -1,9 +1,8 @@
-import { SafeString } from '@glimmer/runtime';
```

Comment:
we can't import from Glimmer, because Glimmer is ESM, and Glint is not.

Reply:
Alternatively, the tests could be extracted to an ESM package, but then we'd lose the ability to test private things.

A solution could be vitest's in-source testing.
This would require revamping how all the glint packages are built though. (not using tsc)